src/Enums/AvailableProfiles.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Enums;
+
+use function __;
+use Illuminate\Support\Collection;
+
+enum AvailableProfiles: string
+{
+    case Facebook = "Facebook";
+    case Linkedin = "Linkedin";
+    case Instagram = "Instagram";
+    case YouTube = "YouTube";
+    case Twitter = "Twitter";
+    case GitHub = "GitHub";
+
+    public function label(): string
+    {
+        return __($this->value);
+    }
+
+    public static function casesToSelect(): array
+    {
+        $cases = AvailableProfiles::all();
+
+        return $cases->map(
+            fn(AvailableProfiles $enum) => [
```

Comment:
```suggestion
            fn(AvailableProfiles $enum): array => [
```

Anonymous functions should be typed.